drivers/gpio/gpio_smartbond.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2022, Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_smartbond_gpio
+
+#include <stdint.h>
+#include <zephyr/drivers/gpio.h>
+#include <DA1469xAB.h>
+#include "gpio_utils.h"
```

Comment:
```
#include "gpio_utils.h"

#include <stdint.h>
#include <zephyr/drivers/gpio.h>

#include <DA1469xAB.h>
```

Reply:
I cannot change order as proposed as this will not build - `gpio_utils.h` does not include required headers, e.g. `stdint.h`

Reply:
FYI https://github.com/zephyrproject-rtos/zephyr/pull/49118